OasisDEX/oasis-borrow
# Added function to check for liquidated vault and added a way of rende…

# [Protection Active Status added](https://app.shortcut.com/oazo-apps/story/5321/remove-protections-active-status-on-overview-after-stop-loss-is-triggered)
    
## How to test 🧪
Use http://localhost:3000/owner/0x497CB171dDF49af82250D7723195D7E47Ca38A95?network=goerli#multiply and you should see that vaults that have been liquidated show as active status
